app/models/menu.rb

```diff
@@ -1,4 +1,6 @@
 class Menu < ApplicationRecord
   belongs_to :user
-  validates :name presence: true
+  validates :name, :photo, :price, :preparation_time, :min_guests, :max_guests, :category, presence: true, uniqueness: true
```

Comment:
don't use uniquess for min and max guests. It would mean that only one meal can have 4 guests, whereas several meals can have the same number of servings